src/workers/saveEconomy.ts

```diff
@@ -0,0 +1,55 @@
+import { DiscordJob, DiscordWorker } from '../lib/DiscordWorker'
+import { apiFetch } from '../lib/api'
+import { defaultMetadata, formatAsReportingPeriods } from '../lib/saveUtils'
+import redis from '../config/redis'
+import { getReportingPeriodDates } from '../lib/reportingPeriodDates'
+
+export class JobData extends DiscordJob {
+  declare data: DiscordJob['data'] & {
+    companyName: string
+    wikidata: any
+    fiscalYear: any
+    economy?: any[]
+  }
+}
+
+const saveEconomy = new DiscordWorker<JobData>(
+  'saveEconomy',
+  async (job) => {
+    const { url, fiscalYear, wikidata, economy = [] } = job.data
+    const wikidataId = wikidata.node
+    const metadata = defaultMetadata(url)
+
```

Comment:
Economy is newer checked for diff. We want to make sure we don't overwrite existing data.